cors/generate.go

```diff
@@ -178,8 +189,16 @@ func {{ .Endpoint.MountHandler }}(mux goahttp.Muxer, h http.Handler) {
 var handleCORST = `{{ printf "%s applies the CORS response headers corresponding to the origin for the service %s." .OriginHandler .Name | comment }}
 func {{ .OriginHandler }}(h http.Handler) http.Handler {
 {{- range $i, $policy := .Origins }}
+	{{- if $policy.EnvVar }}
+	originStr{{$i}}, present := os.LookupEnv({{ printf "%q" $policy.Origin }})
+	if !present {
+		panic("CORS origin environment variable \"{{ $policy.Origin }}\" not set!")
+	}
+	{{- else }}
+	originStr{{$i}} := {{ printf "%q" $policy.Origin }}
+	{{- end }}
 	{{- if $policy.Regexp }}
-	spec{{$i}} := regexp.MustCompile({{ printf "%q" $policy.Origin }})
```

Comment:
Would it be possible to test and only introduce the local variable if using an env var? The overall philosophy of Goa has always been to optimize for generated code readability / idiomatic Go rather than ease of templating.

Reply:
I see what you mean! I just made a change to just use a literal if EnvVar isn't set. Let me know what you think.